web/src/app/dialogs/new-inspection/components/service/parameter-store.ts

```diff
@@ -88,8 +93,11 @@ export class DefaultParameterStore implements ParameterStore {
         take(1),
       )
       .subscribe((parameters) => {
-        parameters[id] = value;
-        this.currentParameters.next(parameters);
+        this.currentParameters.next({
+          ...parameters,
+          [id]: value,
+        });
+        console.log('parameter update', value);
```

Comment:
Is it intended to keep this log?

Reply:
Thanks. Removed.
I'm thinking if there were good way to check this with linter.

Reply:
Maybe we can use ESLint "no-console" rule and `ng lint` ?
https://eslint.org/docs/latest/rules/no-console
https://angular.jp/cli/lint